src/uwtools/resources/info.json

```diff
@@ -0,0 +1,4 @@
+{
+  "version": "2.1.1",
+  "buildnum": "0"
+}
```

Comment:
This should now be the single source of knowledge for version and build number in the `uwtools` repo. (`recipe/meta.json` also contains this information, but it it automatically generated by running `make meta` and ultimate gets its copies of these values from this file.)